### PR TITLE
Add ability to seed multiple default users with configuration binding

### PR DESCRIPTION
**Pull request by Kronos11**
This was a nice to have for our team as we wanted our whole team to have access to setup Toggles

### CODE REVIEW

**Comment by unaizorrilla** [comment]
Hi @Kronos11 

As always! thanks for contribute this! I try to review asap!

File: src/Esquio.UI.Host/Startup.cs [hunk 1]
```diff
@@ -51,6 +52,9 @@ public void ConfigureServices(IServiceCollection services)
                 .AddJwtBearer(options =>
                 {
                     Configuration.Bind("Security:OpenId", options);
+
```
[the remark this answers is not in the document]
Comment (Kronos11) [reply src/Esquio.UI.Host/Startup.cs:56]: Will do

File: src/Esquio.UI.Host/appsettings.json [hunk 2]
```diff
@@ -1,47 +1,67 @@
...
-        "Args": {
-          "outputTemplate": "[{Timestamp:yyyy-MM-dd HH:mm:ss.fff zzz} {Level} {MachineName} {SourceContext} {Message} {Exception} {NewLine}"
+    },
+    "AllowedHosts": "*",
+    "Cors": {
+        "Origins": "*" // allowed origins for cors separated by , character use '*' to allow all origins
+    },
+    "Security": {
+        "DefaultUsers": [{
+                "ApplicationRole": "Management",
```
[the remark this answers is not in the document]
Comment (Kronos11) [reply src/Esquio.UI.Host/appsettings.json:31]: Yup

**Comment by Kronos11** [comment]
@unaizorrilla update per comments

**Comment by unaizorrilla** [comment]
Merged, thanks for contribute!

I do some minor changes like:

 - Only one way to specify default subject
 - Modify demo's docker-compose with new default users configuration section

**Pull request merged**